sklearn/ensemble/forest.py

```diff
@@ -1631,6 +1631,19 @@ class labels (multi-output problem).
     reduce memory consumption, the complexity and size of the trees should be
     controlled by setting those parameter values.
 
+    Examples
+    --------
+    >>> from sklearn.ensemble import ExtraTreesClassifier
+    >>> from sklearn.datasets import make_classification
+    >>> X, y = make_classification(n_features=4, random_state=0)
```

Comment:
what's your reasoning for using different random states?

Reply:
Actually, there is no logic for it. I think it doesn't make any sense either. Therefore, will be changing the code.

Reply:
I have made the changes, _linter test is not getting passed_ but it is not due to my code changes.